ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/resources/impl/ProjectImpl.java

```diff
@@ -96,6 +97,11 @@ public SourceStorage getSource() {
     return reference.getSource();
   }
 
+  @Override
+  public List<ProjectProblem> getProblems() {
+    return null;
```

Comment:
Maybe it would be better to return actual list of problems? `reference.getProblems()` WDYT?

Reply:
yes, thx